pkg/util/utils_test.go

```diff
@@ -70,8 +70,8 @@ func TestValidateProxyURI(t *testing.T) {
 		"https://foo:b@r@test.com:454": true,
 	}
 	for uri, val := range urlList {
-		if ValidateProxyURI(uri) != val {
-			t.Fatalf("Expected '%t' Got '%t'", val, ValidateProxyURI(uri))
+		if ValidProxyURI(uri) != val {
```

Comment:
urlList should check the cases described here:

```
      --http-proxy string               HTTP proxy for virtual machine (In the format of http://<username>:<password>@<proxy_host>:<proxy_port>)
      --https-proxy string              HTTPS proxy for virtual machine (In the format of https://<username>:<password>@<proxy_host>:<proxy_port>)
```

this means, `username:password@` should be tested for